lib/trace-writer.js

```diff
@@ -74,6 +74,7 @@ TraceWriter.prototype.writeSpan = function(spanData) {
   if (this.config_.instanceId) {
     spanData.addLabel(traceLabels.GCE_INSTANCE_ID, this.config_.instanceId);
   }
+  spanData.addLabel(traceLabels.AGENT_DATA, 'node ' + pjson.version);
```

Comment:
The name currently is "@google/cloud-trace" which doesn't carry any language specific information (and I don't think it necessarily should). The fact that this is a trace agent can be inferred from the fact that we're reading the information from a trace span.